app/lib/features/comments/widgets/comment_list_widget.dart

```diff
@@ -42,6 +42,7 @@ class CommentListWidget extends ConsumerWidget {
       shrinkWrap: shrinkWrap,
       itemCount: commentList.length,
       padding: EdgeInsets.zero,
+      reverse: true,
```

Comment:
just to then reverse the order here ...

but I suspect this is due to the scroll-position?

Reply:
I found this as better solution then having scroll controller and scroll to last.

Note:
ListView and List Provider are generic so don't think so there will be any issue with this approach.

Happy to change this later if found better solution.